### ISSUE TITLE
Wiki images

### ISSUE THREAD

**Issue by joha2** [mess42/pyrate#21]
if you load up images right here, you can link to them in the Wiki.

**Comment by joha2** [comment]
![pyrate_cosy1](https://cloud.githubusercontent.com/assets/12564815/19412519/aaca63ca-9318-11e6-84d3-23cddbde6009.png)
![pyrate_cosy2](https://cloud.githubusercontent.com/assets/12564815/19412520/aacfa394-9318-11e6-813a-70b215059202.png)
![pyrate_cosy3](https://cloud.githubusercontent.com/assets/12564815/19412521/aad85994-9318-11e6-82c9-1727af5374b2.png)
![pyrate_cosy4](https://cloud.githubusercontent.com/assets/12564815/19412522/aada7e54-9318-11e6-9412-4699c774d57e.png)

**Comment by joha2** [comment]
![pyrate_gui_20_fieldp](https://cloud.githubusercontent.com/assets/12564815/20868818/1c4fcd32-ba65-11e6-92c5-370449155fe6.png)

**Comment by joha2** [comment]
![achromat](https://cloud.githubusercontent.com/assets/12564815/24621643/81348620-18a2-11e7-9d04-4a6c8c591c16.png)

**Comment by HoWilgh** [comment]
Looking really nice now!

**Comment by joha2** [comment]
![pyrate_mirrors_comp_parabasal_exact](https://cloud.githubusercontent.com/assets/12564815/24820302/9b8cf4a0-1be8-11e7-8d8b-de0184587145.png)

**Comment by joha2** [comment]
![pyrate_prism](https://cloud.githubusercontent.com/assets/12564815/24830990/008aa782-1c91-11e7-8d7b-749fe6b99a61.png)

**Comment by joha2** [comment]
![pyrate_grin](https://cloud.githubusercontent.com/assets/12564815/24837010/87ed06ae-1d2a-11e7-8794-bab44d618e4b.png)

**Comment by luzpaz** [comment]
Fantastic!
Can you please add some of these to the README?

**Comment by joha2** [comment]
@luzpaz yes we will add some of these to the readme. But for us it was more important that they are available in the wiki.

**Comment by joha2** [comment]
![pyrate_rainbow](https://user-images.githubusercontent.com/12564815/28295078-7fcfed9c-6b5e-11e7-8710-13fce405dfe0.png)

**Comment by joha2** [comment]
![grin_1](https://user-images.githubusercontent.com/12564815/40586439-3adda2f0-61c2-11e8-89b7-ee88b96ff7d4.png)
![grin_2](https://user-images.githubusercontent.com/12564815/40586440-3b027a80-61c2-11e8-89b3-01572ace57e3.png)
![grin_spot](https://user-images.githubusercontent.com/12564815/40586441-3b23f1d8-61c2-11e8-87e6-e17fec4490fb.png)
